starter_code/script.js

```diff
@@ -4,26 +4,52 @@ var aiPoint = 0;
 // This function returns the selection of the computer
 function getAISelection() {
     //TODO: randomly choose between 'rock', 'paper', or 'scissors'
+    var random = Math.random();
+    if(random < 1/3)
+    {
+        return 'rock';
+    }
+    if(random < 2/3)
+    {
+        return 'paper';
+    }
+        return 'scissors';
 }
 
 // This function picks the winner
 function pickWinner(userValue, aiValue) {
     //TODO: pick the correct winner: user or ai
     //TODO: Add one point for the winner
+    if(userValue === aiValue)
+    {
+        return 'draw';
+    }
+    
+    if(userValue === 'rock' && aiValue === 'paper' ||
+       userValue === 'paper' && aiValue === 'scissors'||
+       userValue === 'scissors' && aiValue === 'rock')
+    {
+        aiPoint += 1;
+        return 'ai'
+    }
+    userPoint += 1;
+    return 'user'
 }
 
 // This function sets the scoreboard with the correct points
 function setScore() {
-
+    $('#userPoint').text(userPoint);
+    $('#aiPoint').text(aiPoint);
 }
 
 // This function captures the click and picks the winner
 function evaluate(evt) {
+    debugger
```

Comment:
did that work alright for you? if not, make sure to have your browser's console open when you run the function!